clients/python/tests/regression_test.py

```diff
@@ -22,3 +22,25 @@ def test_create_tagged_version(client: ModelRegistry):
     mv = client.get_model_version(name, version)
     assert mv
     assert mv.id
+
+@pytest.mark.e2e
+def test_get_model_without_user_token(setup_env_user_token, client):
```

Comment:
I see here you're already using it. So the scope reduction is a better practice.